server/lib/currency.ts

```diff
@@ -264,3 +264,13 @@ export function reduceArrayToCurrency(array: AmountWithCurrencyAndDate[], curren
     },
   );
 }
+
+/**
+ * Gets exchange rate between given crypto currency and fiat.
+ */
+export async function exchangeRateQuery(cryptoCurrency: string, fiatCurrency: string): Promise<string> {
+  if (!cryptoCurrency || !fiatCurrency) {
+    return null;
+  }
+  return getFxRate(cryptoCurrency, fiatCurrency);
+}
```

Comment:
I don't see what this function is doing.

Reply:
Sorry about this, I initially thought that we could simply use `getFxRate` and made a commit like this; but later reverted it back to my original function; please see the above comment; https://github.com/opencollective/opencollective-api/pull/7541#discussion_r876427807